projects/plugins/crm/sass/_ZeroBSCRM.prevadmincore.scss

```diff
@@ -754,7 +751,6 @@ select[multiple],select[size] {
 }
 
 input[type="checkbox"]:focus,input[type="file"]:focus,input[type="radio"]:focus {
-    outline:thin dotted;
     outline:5px auto -webkit-focus-ring-color;
```

Comment:
I note that this change may have an effect in browsers that don't recognize `-webkit-focus-ring-color`. Previously they'd get `outline:thin dotted`, while now they'll inherit styles from somewhere else.

OTOH, on Firefox 137, the inherited styles seem like they'll be ok. Focus from click won't be indicated, but focus via keyboard tabbing will be more obvious. 🤷

Reply:
These weren't intentional. I'd accidentally run `--fix` with `declaration-block-no-duplicate-properties` fully enabled at one point, which chopped all the duplicate fallback properties, and I thought I restored them all, but apparently missed a few.

I'm keeping that out of scope and have restored that line: 1594f7c3d5